src/integrationTest/java/uk/gov/hmcts/cmc/claimstore/controllers/SaveClaimWithCoreCaseDataStoreTest.java

```diff
@@ -25,7 +25,7 @@
         "document_management.url=false",
         "feature_toggles.ccd_async_enabled=false",
         "feature_toggles.ccd_enabled=true",
-        "feature_toggles.async_event_operations_enabled=false"
+        "feature_toggles.async_event_operations_enabled=true"
```

Comment:
Is this required and related to this ?

Reply:
yeah .. if this is not true then it triggers the old claimissueevent. IMHO where ever applicable we need to turn on feature_toggles.async_event_operations_enabled=true